spec/spec_helper_acceptance.rb

```diff
@@ -84,9 +81,7 @@ def setup_puppet_on(host, opts = {})
   opts = {:agent => false, :mcollective => false}.merge(opts)
 
   step "Setup puppet on #{host}"
-  install_package host, 'puppet'
-  add_foss_defaults_on host
-  add_puppet_paths_on host
+  install_puppet_on host
```

Comment:
All the `step` methods should be removed, but I hadn't heard they were officially yanked from beaker-rspec.

Anything that needs to be added for Windows would happen under PUP-4479.